index.html

```diff
@@ -36,98 +36,95 @@ <h2>Brigade is a tool for running scriptable, automated tasks in the cloud &mdas
         <div class="medium-7 columns">
 
           <h2>Simple, powerful pipes</h2>
-          <p>Each project gets a <code>brigade.js</code> config file, which is where you can write dynamic, interwoven pipelines and tasks for your Kubernetes cluster. Leave your YAML at home!</p>
+          <p>Each project uses a script file written in either JavaScript or TypeScript, which is where you can write dynamic, interwoven pipelines and tasks to handle inbound events. Leave your YAML at home!</p>
 
           <ul class="example-orbit-content" data-orbit>
             <li data-orbit-slide="headline-1">
               <pre>
-<em>// Run unit tests for a Github push</em>
-const { events, Job , Group} = require("brigadier");
-const dest = "$GOPATH/src/github.com/technosophos/ulid";
-
-events.on("push", (e, p) => {
-  console.log(e.payload)
-  var gh = JSON.parse(e.payload)
-  var test = new Job("test", "golang:1.9")
-  test.tasks = [
-    "mkdir -p " + dest,
-    "cp -a /src/* " + dest,
-    "cd " + dest,
-    "go get -u github.com/golang/dep/cmd/dep",
-    "dep ensure",
-    "make test"
-  ];
-  test.run()
-});
+<em>// Run unit tests for a Github push using TypeScript</em>
+import { events, Event, Job } from "@brigadecore/brigadier"
+
+const localPath = "/workspaces/brigade"
+
+events.on("brigade.sh/github", "push", async event => {
+  let test = new Job("test", "debian:latest", event)
+  test.primaryContainer.sourceMountPath = localPath
+  test.primaryContainer.workingDirectory = localPath
+  test.primaryContainer.command = ["bash"]
+  test.primaryContainer.arguments = ["-c", "make test"]
```

Comment:
This could be shortened / mad simpler:

```
test.primaryContainer.command = ["make"]
test.primaryContainer.arguments = ["test"]
```

Reply:
Also, while I love this example, it doesn't actually work because `debian:latest` does not come with `make` already installed.

Reply:
Used a golang image, which does have make 🎉